src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java

```diff
@@ -91,15 +87,18 @@ public static double get(final double difference) {
 
         private static void initialize() {
             if (cache == null) {
-                final int tableSize = (int) (MAX_TOLERANCE / TABLE_STEP) + 1;
+                final int tableSize = (int) (MAX_TOLERANCE / TABLE_STEP) + 2;
```

Comment:
This was an artifact of out-of-bounds-by-1 error in the unit tests that arose from using truncated values of log(10).  Now that I have the exact calculation it can go back to how it was.